spec/models/card_spec.rb

```diff
@@ -0,0 +1,32 @@
+require 'rails_helper'
+
+RSpec.describe Card, type: :model do
+  it "should correct verify compliance translations" do 
+    test_card = Card.create!(original_text: "Mom wash frame", translated_text: "Мама мыла раму")
+
+    expect(test_card.check_translation("мАмА  МыЛА   РамУ")).to eq(true)
+  end
+
+    it "should correct verify compliance translations" do 
+    test_card = Card.create!(original_text: "Mom wash frame", translated_text: "Мама мыла раму")
+
+    expect(test_card.check_translation("Мать протирала окно")).to eq(false)
+  end
```

Comment:
`end` at 14, 2 is not aligned with `it "should correct verify compliance translations" do ` at 10, 4